deepchem/models/torch_models/gan.py

```diff
@@ -716,14 +716,8 @@ def __init__(self,
         """
         self.n_generators = n_generators
         self.n_discriminators = n_discriminators
+        self.device = device  # type: ignore
```

Comment:
This needs to be changed to be beginner friendly; a beginner with a GPU should have it used by default

Reply:
fixed